HW10/Weather-API.py

```diff
@@ -0,0 +1,18 @@
+import requests
+
+API = '84b66753dae64199592e1bedb91f9b9a'
+def get_weather(city: str) -> dict:
+    response = requests.get(f'https://api.openweathermap.org/data/2.5/weather?q={city}&appid={API}')
```

Comment:
The part https://api.openweathermap.org/data/2.5/ would be better move to constant BASE_URL